.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+sudo: false
+python:
+  - "2.7"
+env:
+  - TOX_ENV=docs
+install:
+  - pip install -rrequirements/docs.txt
+script:
+  - cd docs && make html SPHINXOPTS='-nW'
```

Comment:
Yeah I was thinking of linkcheck [1] 

[1] http://sphinx-doc.org/invocation.html#invocation-of-sphinx-build